libs/sdk-ui-kit/src/Form/InputPure.tsx

```diff
@@ -23,8 +23,8 @@ export interface InputPureProps {
     onChange: (value: string | number, e?: React.ChangeEvent<HTMLInputElement>) => void;
     onEscKeyPress: () => void;
     onEnterKeyPress: () => void;
-    onBlur: () => void;
-    onFocus: () => void;
+    onBlur: (e?: React.FocusEvent<HTMLInputElement>) => void;
```

Comment:
Is this argument really optional? Can the handler be called without the event present? Ditto for the following

The problem with this is that if the user wants to access the event, they would need to use `e?.target` for example, even though the `e` is aways there.